.freeCodeCamp/tooling/locales/english/build-a-smart-contract-in-rust.md

```diff
@@ -6,9 +6,9 @@
 
 You will build a Smart Contract using Rust.
```

Comment:
```suggestion
For this project, you need to create and deploy Smart Contract using Rust that keeps track of how many users have clicked a button on your web page.
```

This isn't a specific suggestion - but a suggestion to make it more specific. Something to let users know a little more about what they will build. I added `For this project` at the start - one of my projects has that, two of them don't. I was thinking we could maybe add it to all of them.